docs/dev-guide.md

```diff
@@ -67,8 +67,8 @@ Here's a quick checklist to follow when coding to ensure a good pull request
   goal.
 - Ensure all indentation is done as 4-spaces and your editor is set to unix line
   endings.
-- The code matches PEP8 style guides. Make sure to run
-  `pycodestyle tableauserverclient test samples` to catch and fix any style
+- The code matches PEP8 style guides, enforced by the black formatter. Make sure to run
+  `black --check --line-length 120 tableauserverclient samples test` to catch and fix any style
```

Comment:
Add a mention that it will auto-fix the code if you leave out the --check parameter